src/controller/controller.cpp

```diff
@@ -9,7 +9,7 @@ void Controller::init() {
     static Uniform distY = Uniform(0.0, world.HEIGHT);
     static Uniform distE = Uniform(MIN_FOOD_SIZE, MAX_FOOD_SIZE);
 
-    for (int i = 0; i < 10; i++) world.addIndividual(new Fred(10 * i, 10 * i, 40));
```

Comment:
orientation should be random at spawn I'd say

Reply:
I thought so at first too, but since we decided that the controller initialises everything, shouldn't the direction also be initialised explicitly by it?

Reply:
yes